src/main/java/com/linecorp/armeria/server/http/jetty/JettyService.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.http.jetty;
+
+import static java.util.Objects.requireNonNull;
+
+import org.eclipse.jetty.server.Server;
+
+import com.linecorp.armeria.server.http.HttpService;
+
+public class JettyService extends HttpService {
+
+    public static JettyService forServer(String hostname, Server server) {
```

Comment:
Would name the variable jettyServer to better differentiate from an armeria server (which is what I thought at first glance).